examples/rust-key-value/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name    = "spin-key-value"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = [ "cdylib" ]
+
+[dependencies]
+# Useful crate to handle errors.
+anyhow = "1"
+# Crate to simplify working with bytes.
+bytes = "1"
+# General-purpose crate with common HTTP types.
+http = "0.2"
+spin-sdk = { path = "../../sdk/rust"}
+# The wit-bindgen-rust dependency generates bindings for interfaces.
+wit-bindgen-rust = { git = "https://github.com/bytecodealliance/wit-bindgen", rev = "cb871cfa1ee460b51eb1d144b175b9aab9c50aba" }
+
+[workspace]
+
+# Metadata about this component.
+[package.metadata.component]
+name = "spinhelloworld"
```

Does this metadata need to be changed to match the component name in the `spin.toml`?
```suggestion
name = "hello"
```

---

I had just cut-and-pasted that from another example.  I've removed the section entirely since it doesn't seem to do anything.